From e56e39f90855cb5e988bd35130bc57d18cb9d9f6 Mon Sep 17 00:00:00 2001
From: Pascal Vizeli <pvizeli@syshack.ch>
Date: Wed, 19 Jun 2019 11:43:16 +0000
Subject: [PATCH 1/1] Fix ARM builds

---
 setup.py | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 65322db..447b898 100644
--- a/setup.py
+++ b/setup.py
@@ -117,7 +117,9 @@ def main():
         "-DBUILD_SHARED_LIBS=OFF",
         "-DBUILD_TESTS=OFF",
         "-DBUILD_PERF_TESTS=OFF",
-        "-DBUILD_DOCS=OFF"
+        "-DBUILD_DOCS=OFF",
+        "-DENABLE_NEON=OFF",
+        "-DCPU_BASELINE_REQUIRE="
     ] + (["-DOPENCV_EXTRA_MODULES_PATH=" + os.path.abspath("opencv_contrib/modules")] if build_contrib else [])
 
     # OS-specific components
-- 
2.20.1

